src/main/java/teammates/ui/request/FeedbackSessionUpdateRequest.java

```diff
@@ -1,8 +1,54 @@
 package teammates.ui.request;
 
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * The update request of a feedback session.
  */
 public class FeedbackSessionUpdateRequest extends FeedbackSessionBasicRequest {
 
+    private Map<String, Long> studentDeadlines;
+    private Map<String, Long> instructorDeadlines;
+
+    private boolean isGoingToNotifyAboutDeadlines;
```

Comment:
This file was added from another PR to generate types for the frontend. Tagging @jayasting98 for visibility

Reply:
Just to keep everyone in the loop, after discussion with jay we felt it's best to move this out of the request body and send this as a query param instead. This will be removed in his PR!

https://github.com/TEAMMATES/teammates/pull/11649#discussion_r830504851

Reply:
>Seems a bit lengthy

Yea I was trying to make it start with "is". Samuel has told me to shorten it too, which I will do.